doc/README.msys2

```diff
@@ -74,18 +68,18 @@ using MSYS2 from http://msys2.github.io
  - mingw-w64-i686-imagemagick / mingw-w64-x86_64-imagemagick
  - mingw-w64-i686-SDL2_mixer / mingw-w64-x86_64-SDL2_mixer
 
-2.1.2.1.2) Arch-specific optional packages for building common parts
+2.1.2.1.2) Arch-specific optional packages for building common parts (i686 or x86_64, not both)
  - mingw-w64-i686-drmingw / mingw-w64-x86_64-drmingw
  - mingw-w64-i686-meson / mingw-w64-x86_64-meson
```

Comment:
There are some outdated deps here, but maybe you want to do it in a second pass?

Reply:
Fixed in third commit